integration-tests/sdk/auth.it.js

```diff
@@ -23,6 +23,43 @@ function getApiClient(token) {
   })
 }
 
+function modifiedTokenInfo(tokenInput, unUsedParameter) {
+  const receivedScope = tokenInput.scope
+  let changeScopeToArrayOfObject = '[{'
+  const newScope = receivedScope.split(' ')
+
+  for (let i = 0; i < newScope.length; i++) {
+    const splitScope = newScope[i].split(':')
+    changeScopeToArrayOfObject +=
+      '"' + splitScope[0] + '":"' + splitScope[1] + '",'
```

Comment:
unless we are using [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) + [Object.values](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object) + `JSON.stringify`, this is black magic 🔮

Reply:
the point I'm making is that you should aim to incorporate a stategy of how you work with the scopes and have readable code. I encourage a combination of different native Array/Object functions to accomplish that

Reply:
i'm confused.. this could be the same as `changeScopeToArrayOfObject = newScope[i]`

Reply:
It's not black magic, check the code update.